src/Marks/Mark.tsx

```diff
@@ -13,22 +13,30 @@ export interface MarkProps {
 
 export default function Mark(props: MarkProps) {
   const { prefixCls, style, children, value, onClick } = props;
-  const { min, max, direction, includedStart, includedEnd, included } =
+  const { min, max, direction, includedStart, includedEnd, included, styles, classNames } =
     React.useContext(SliderContext);
 
   const textCls = `${prefixCls}-text`;
+  const active = included && includedStart <= value && value <= includedEnd;
 
   // ============================ Offset ============================
   const positionStyle = getDirectionStyle(direction, value, min, max);
 
   return (
     <span
-      className={classNames(textCls, {
-        [`${textCls}-active`]: included && includedStart <= value && value <= includedEnd,
-      })}
+      className={cls(
+        textCls,
+        {
+          [`${textCls}-active`]: active,
+        },
+        active && classNames.markActive,
+        classNames.mark,
+      )}
       style={{
         ...positionStyle,
         ...style,
+        ...(active ? styles.markActive : {}),
+        ...styles.mark,
```

Comment:
This should be:
```javascript
...styles.mark,
...(active ? styles.markActive : {}),
```
`markActive` styles should override the default `mark` styles, not the other way around.